controllers/tmf-apis/party.js

```diff
@@ -66,15 +67,16 @@ var party = (function() {
             });
         } else {
             // regexResult[3] contains the user id
-	    var bod = JSON.parse(req.body)
-            if (req.user.id === regexResult[3] || bod.id === regexResult[3]) {
+	    var org = req.user.organizations.find( x => x.id === regexResult[3]);
+	    // the first condition checks if the individual is the one calling
+	    if (req.user.id === regexResult[3] || org.roles.findIndex(x => x.name === "Admin") > -1) {
```

Comment:
If you are trying to update an organization, and you are not member, org will be undefined, so org.roles.find will give you an exception